plugins/snowflake/setup.py

```diff
@@ -47,11 +47,8 @@
     },
     install_requires=[
         'dbt-core=={}'.format(package_version),
-        'snowflake-connector-python[secure-local-storage]==2.3.6',
-        'azure-common<2.0.0',
-        'azure-storage-blob>=12.0.0,<13.0.0',
```

Comment:
I don't think these are needed. We don't directly use these packages in the codebase to my knowledge and they are have the same constraints in `snowflake-connector-python`
https://github.com/snowflakedb/snowflake-connector-python/blob/77ad07f9a07991719258d6b6c6ebfb48691e51de/setup.py#L197-L198

Reply:
Looks like we initially pinned these back in Nov 2019 (https://github.com/fishtown-analytics/dbt/pull/1878), when we started being much stricter with SCP

I agree, we don't need them, and we wouldn't want to have them if we're loosening SCP from a tight pin